app/controllers/records_controller.rb

```diff
@@ -100,12 +100,21 @@ def find_records_json
 
     response_array = []
     response_records.each do |record|
-      response_array.push({"id":record.id, "state":record.state, "concept_id": record[:concept_id],
-        "date_ingested": record[:date_ingested], "format": record.format,
+      if record.format == 'umm_json'
```

Comment:
I think of instead of repeating the entire "push", just create a variable called format:
format = record.format == 'umm_json' ? 'umm-c' : record.format
Then just call push with "format" variable instead.